app/views/hyrax/base/_form_permission.html.erb

```diff
@@ -0,0 +1,36 @@
+<% # This is used by works and by FileSet and the layout (col-6 vs col-12) is different for both %>
+<% if f.object.embargo_release_date %>
+  <%= render 'form_permission_embargo', f: f %>
+<% elsif f.object.lease_expiration_date %>
+  <%= render 'form_permission_lease', f: f %>
+<% else %>
+  <fieldset class="set-access-controls">
+    <legend>
+      Visibility
+      <small>Who should be able to view or download this content?</small>
+    </legend>
+
+    <div class="form-group">
+      <label class="radio">
+        <%= f.radio_button :visibility, Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC %>
+        <%= visibility_badge(Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC) %>
+      </label>
+      <label class="radio">
+        <%= f.radio_button :visibility, Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED %>
+        <%= visibility_badge(Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED) %>
+      </label>
+      <label class="radio">
+        <%= f.radio_button :visibility, Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_EMBARGO %>
+        <%= render "form_permission_embargo", f: f %>
+      </label>
+      <label class="radio">
+        <%= f.radio_button :visibility, Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_LEASE %>
+        <%= render "form_permission_lease", f: f %>
+      </label>
```

Comment:
There is a bug (I think) in Hyrax where it applies a lease visibility to a file set if any of the lease parameters are not null; regardless of the value of the visibility param. The default lease dates in this form trigger the behavior, so file sets end up always having a lease. When I get a moment, I'll work up a PR to Hyrax about this. In the meantime, since we aren't using them in Plum yet, I think we can remove the lease and embargo sections here. Doing so fixes the issue.